xbmc/interfaces/python/swig.h

```diff
@@ -146,6 +146,8 @@ namespace PythonBindings
   void registerAddonClassTypeInformation(const TypeInfo* classInfo);
   const TypeInfo* getTypeInfoForInstance(XBMCAddon::AddonClass* obj);
 
+  int dummy_tp_init(PyObject *self, PyObject *args, PyObject *kwds);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

Reply:
fixed